xbmc/cores/VideoPlayer/IVideoPlayer.h

```diff
@@ -69,6 +69,14 @@ struct SStateMsg
   int player;
 };
 
+struct SSubtitleMsg
+{
+  std::string path;
+  std::string name;
+  std::string language;
+  bool activate;
```

Comment:
Out of curiosity: Should this `bool` be explicitly initialized, i.e.:

```suggestion
  bool activate = false;
```

just to avoid cases where `SSubtitleMsg` is used but `activate` is forgotten to be set? :-)